js/interface.js

```diff
@@ -1509,1340 +1582,1494 @@ function validateScreenshots() {
-        debugHtmlPage = _.find(submission.data.previousResults.debugHtmlPage.files, function(file) {
-          return file.contentType === 'text/html';
-        });
-      }
+    publishApp('unsigned').catch(function () {
+      $('.button-unsigned-request').html(initialHtml);
+      $('.button-unsigned-request').prop('disabled', false);
+    });
+  }
 
-      build.id = submission.id;
-      build.updatedAt = ((submission.status === 'completed' || submission.status === 'failed' || submission.status === 'cancelled' || submission.status === 'ready-for-testing' || submission.status === 'tested') && submission.updatedAt) ?
-        moment(submission.updatedAt).format('MMM Do YYYY, h:mm:ss a') :
-        '';
-      build.submittedAt = ((submission.status === 'queued' || submission.status === 'submitted') && submission.submittedAt) ?
-        moment(submission.submittedAt).format('MMM Do YYYY, h:mm:ss a') :
-        '';
-      build[submission.status] = true;
-      build.fileUrl = appBuild ? appBuild.url : '';
+  // Gives time to Validator to apply classes
+  setTimeout(checkGroupErrors, 0);
+});
 
-      if (submission.result.message) {
-        build.message = submission.result.message;
-      }
+/* SAVE PROGRESS CLICK */
+$('[data-app-store-save]').on('click', function () {
+  saveAppStoreData();
+});
+$('[data-enterprise-save]').on('click', function () {
+  saveEnterpriseData();
+});
+$('[data-unsigned-save]').on('click', function () {
+  saveUnsignedData();
+});
+$('[data-push-save]').on('click', function () {
+  savePushData();
+});
 
-      if (userInfo.user && (userInfo.user.isAdmin || userInfo.user.isImpersonating)) {
-        build.debugFileUrl = debugHtmlPage ? debugHtmlPage.url : '';
-      }
+/* Credentials and Certificates App Store */
+$('.login-appStore-button').on('click', function () {
+  var $this = $(this);
+  $(this).html('Logging in ' + spinner);
+  $(this).addClass('disabled');
+  var devEmail = $('#fl-store-appDevLogin').val();
+  var devPass = $('#fl-store-appDevPass').val();
+  var emailError = $('#fl-store-appDevLogin').data('error');
+  var passError = $('#fl-store-appDevPass').data('error');
 
-      buildsData.push(build);
-    });
+  // Remove errors
+  $('#fl-store-appDevLogin').parents('.form-group').removeClass('has-error has-danger');
+  $('#fl-store-appDevLogin').next('.with-errors').html('');
+  $('#fl-store-appDevPass').parents('.form-group').removeClass('has-error has-danger');
+  $('#fl-store-appDevPass').next('.with-errors').html('');
+  $this.nextAll('login-error').html('');
 
-    compileStatusTable(true, origin, buildsData);
-  } else {
-    compileStatusTable(false, origin);
+  if (devEmail === '') {
+    $('#fl-store-appDevLogin').parents('.form-group').addClass('has-error has-danger');
+    $('#fl-store-appDevLogin').next('.with-errors').html(emailError);
+    $(this).html('Log in');
+    $(this).removeClass('disabled');
+    Fliplet.Widget.autosize();
   }
-}
-
-
```

Comment:
what's this for?

Reply:
I needed some personal space. Please, just back off.

:trollface: